### PR TITLE
Milder language on README

### PR DESCRIPTION
**Pull request by pkoch**
As per https://github.com/pytest-dev/pytest-timeout/issues/124#issuecomment-1155633546

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -19,13 +19,15 @@ pytest-timeout
 .. |pre-commit| image:: https://results.pre-commit.ci/badge/github/pytest-dev/pytest-timeout/master.svg
    :target: https://results.pre-commit.ci/latest/github/pytest-dev/pytest-timeout/master
 
-**This is not the timeout you are looking for!**
+To catch tests that take much more time than what you expect them to.
```
Comment (flub) [inline README.rst:22]: Let's just remove this line

File: README.rst [hunk 2]
```diff
@@ -19,13 +19,15 @@ pytest-timeout
...
 
 .. warning::
 
    Please read this README carefully and only use this plugin if you
-   understand the consequences.  Remember your test suite needs to be
-   **fast**, timeouts are a last resort not an expected failure mode.
+   understand the consequences.  This plugin was designed to catch
+   gross, not precise, and unexpected timings.  Remember your test
```
Comment (flub) [inline README.rst:28]:
```suggestion
   understand the consequences.  This plugin is designed to catch
   excessively long test durations like deadlocked or hanging tests,
   it is not designed for precise timings or performance regressions.
   Remember your test
```

**Review by flub — COMMENTED** [review]
I think this is the right direction, though the main sentence added was somewhat confusingly formulated.  I made some suggestions but feel free to try and further improve on them.

File: README.rst (hunk 1 above)
Comment (pkoch) [reply README.rst:22]:
```suggestion

```

**Comment by pkoch** [comment]
Accepted both suggestions, rejiggered whitespace to look nice.

Ready to merge? Something else missing?

**Pull request merged**